src/Component/Customer/CustomerSelector.php

```diff
@@ -66,7 +66,7 @@ public function get()
         $customer = null;
         $user = null;
 
-        if (true === $this->securityContext->isGranted('IS_AUTHENTICATED_FULLY')) {
+        if (true === $this->securityContext->isGranted('IS_AUTHENTICATED_REMEMBERED')) {
```

Comment:
shouldn't this check both? or if you are `fully` you are also `remembered`?

Reply:
If I understand it correctly you are also `remembered` if you are `fully`:
https://symfony.com/doc/current/security/remember_me.html#forcing-the-user-to-re-authenticate-before-accessing-certain-resources

```
// allow any authenticated user - we don't care if they just
// logged in, or are logged in via a remember me cookie
$this->denyAccessUnlessGranted('IS_AUTHENTICATED_REMEMBERED');
```